src/games/strategy/net/MacFinder.java

```diff
@@ -36,41 +41,26 @@ private static String GetMacAddress() {
       final InetAddress address = InetAddress.getLocalHost();
       final NetworkInterface localHostNI = NetworkInterface.getByInetAddress(address);
       if (localHostNI != null) {
-        // final byte[] rawMac = localHostNI.getHardwareAddress();
-        final Method m = NetworkInterface.class.getMethod("getHardwareAddress");
```

Comment:
The reason for this to be there is according to the comments down below, that this methods exists since java 6... So If the computer does not java 6 this code is skipped. Now that we are requiring java 8 this is no longer necessary